### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by afischer** [nytimes/library#258]
### Problem Description

We should make sure that both the app shell and parsed google doc HTML is accessible to all users. The Library shell and some example pages should be audited to make sure they are screen reader-friendly, the default theme should be checked for web content accessibility guidelines (WCAG) AA compliance (at least), and the correct semantic HTML elements should be used wherever possible.

### Additional Information

Some helpful tools: 

- [Full W3C WCAG guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/) (very comprehensive, for reference)
- [a11y project WCAG checklist](https://www.a11yproject.com/checklist/)
- [Lighthouse](https://developers.google.com/web/tools/lighthouse/) is built into Chrome, which gives an a11y score along with recommendations.
